packages/fuel-gauge/src/transaction-upgrade.test.ts

```diff
@@ -167,7 +167,7 @@ describe('Transaction upgrade state transition', () => {
     }
   });
 
-  it('should correctly upgrade chain with uploaded bytecode', { timeout: 15000 }, async () => {
+  it('should correctly upgrade chain with uploaded bytecode', { timeout: 40_000 }, async () => {
```

Comment:
This I don't remember, did you try with 30s?

Reply:
@arboleya no. I've just use `40_000` to be sure

Reply:
🚢